### ISSUE TITLE
Generator data set does not include MWh capacity of existing storage units 

### ISSUE THREAD

**Issue by fikrikscg** [PowerGenome/PowerGenome#99]
I recently encountered this issue. I may have made some mistakes, but it appears that generator data set does not show MWh capacity of existing storage units even though I activated the relevant row in generator_columns parameter. An example  is shown below:

generator_columns: [
            "region",
            "Resource",
            "cluster",
            "R_ID",
            "zone",
            "STOR",
            "Commit",
            "New_Build",
            "Existing_Cap_MWh" # This does not show up in generator data set
]

**Comment by gschivley** [comment]
@fikrikscg what version of pandas are you using? I have it pinned at 0.25.3, which still lets you slice when columns don't exist. We generally don't have `Existing_Cap_MWh` for existing resources. I'm actually not sure if it's available in any EIA data, but as battery storage grows this will become a bigger issue.

**Comment by fikrikscg** [comment]
I am using 0.25.1. Should I update and try again?

**Comment by gschivley** [comment]
No, that should be fine. I might have misunderstood your original issue. Are you getting an error, or is it just that the column `Existing_Cap_MWh` doesn't show up? The latter makes sense, because I don't think we have a data source in PUDL with the MWh of storage facilities. It looks like EIA 860 does have a file with this info (`3_4_Energy_Storage_Y2019.xlsx`).

As with offshore wind, the amount of existing storage is small enough that this is probably negligible when modeling out a couple decades, but it will quickly become a bigger issue.

In the short term I can add a small data file with the battery energy storage info and access it. The better long-term solution will be to add that info to PUDL. Do you have any appetite for creating a clean csv of the 860 energy storage file and starting a PR that would integrate the info? I can help with planning but have too many other priorities to do this myself right now.

**Comment by fikrikscg** [comment]
I am sorry for long delay in replaying. I had found a data set of existing storage units in the U.S. and calculated average durations (MWh divided by MW) for each resource type (battery, pumped hydro, flywheel, solar thermal, and air storage). I am currently using these durations to multiply them with MW capacity of given storage units in generator data set to get a column of MWh capacity. I am using the following numbers:

Battery: 3.541 hours
Compressed air storage: 15.25 hours
Flywheels: 0.25 hours
Thermal storage: 6 hours
Hydroelectric pumped: 13.607 hours

I have also checked 3_4_Energy_Storage_Y2019.xlsx data file, and created the following average duration table:

Battery: 2.34 hours
Compressed air storage: 1 hour
Flywheels: 0.7 hours

I think it is divergent from the first table, and there is no information about pumped hydro units. Let me continue using the first table.

I am not expert in GitHub procedures, so let me not start a pull request. I do not want to mess up :)

**Comment by fikrikscg** [comment]
By the way, as for your first question, I am not getting any error. MWh capacity column does not show up. It makes sense if we do not have such a data.
